pkg/network/encoding/format.go

```diff
@@ -127,7 +128,10 @@ func FormatConnectionTelemetry(tel map[network.ConnTelemetryType]int64) map[stri
 		return nil
 	}
 
-	ret := make(map[string]int64)
+	// Fetch USM payload telemetry
+	ret := telemetry.ReportPayloadTelemetry("")
```

Comment:
Couple of things here: 
1. this will be skipped if `tel == nil`
2. should we have a separate entry for USM metrics? `Connections.ConnTelemetry` only has a very specific list of telemetry that is exposed. It would be better to have a more deterministic list of metrics that are reported to the backend.

Reply:
1. You're right 🙈 
2. I don't have a strong opinion about having it in a separate field or not, but I don't follow what do you mean by "having a specific set of metrics". It's not that we're sending every USM metric to the backend, we would only send metrics that are explicitly declared with the option `telemetry.OptTelemetry`

Reply:
Like we need a list, e.g.: https://github.com/datadog/datadog-agent/blob/main/pkg/network/event_common.go#L147

Reply:
Done, let me know if things are OK now